fw-child/clone-taxs.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Template Name: Clone Taxonomy
+ *
+ * Script to clone a taxonomy to a new one for a given CPT.
+ *
+ * This script utilizes $_GET parameters:
+ *  - old-tax: Slug of the old taxonomy
+ *  - new-tax: Slug of the new taxonomy
+ *  - cpt: Post type associated with the taxonomies
+ */
+
+if ( isset( $_GET['old-tax'], $_GET['new-tax'], $_GET['cpt'] ) ) {
```

Comment:
To prevent undesired execution of this script, I would also add a check that the user is logged to the admin.